[2025-02-11 15:51:30,719] {scheduler_job.py:182} INFO - Started process (PID=1743) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:51:30,720] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:51:30,720] {logging_mixin.py:104} INFO - [2025-02-11 15:51:30,720] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:51:30,730] {logging_mixin.py:104} INFO - [2025-02-11 15:51:30,729] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dev/test_1/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dev/test_1/dag.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2025-02-11 15:51:30,730] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:51:30,792] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.074 seconds
[2025-02-11 15:52:01,677] {scheduler_job.py:182} INFO - Started process (PID=1773) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:52:01,679] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:52:01,680] {logging_mixin.py:104} INFO - [2025-02-11 15:52:01,680] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:52:01,696] {logging_mixin.py:104} INFO - [2025-02-11 15:52:01,695] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dev/test_1/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dev/test_1/dag.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2025-02-11 15:52:01,697] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:52:01,772] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.100 seconds
[2025-02-11 15:52:52,685] {scheduler_job.py:182} INFO - Started process (PID=44) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:52:52,686] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:52:52,687] {logging_mixin.py:104} INFO - [2025-02-11 15:52:52,687] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:52:52,710] {logging_mixin.py:104} INFO - [2025-02-11 15:52:52,709] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dev/test_1/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dev/test_1/dag.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2025-02-11 15:52:52,711] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:52:52,791] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.109 seconds
[2025-02-11 15:53:23,289] {scheduler_job.py:182} INFO - Started process (PID=74) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:53:23,291] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:53:23,292] {logging_mixin.py:104} INFO - [2025-02-11 15:53:23,292] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:53:23,304] {logging_mixin.py:104} INFO - [2025-02-11 15:53:23,303] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dev/test_1/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dev/test_1/dag.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2025-02-11 15:53:23,305] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:53:23,378] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.091 seconds
[2025-02-11 15:53:53,510] {scheduler_job.py:182} INFO - Started process (PID=103) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:53:53,511] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:53:53,511] {logging_mixin.py:104} INFO - [2025-02-11 15:53:53,511] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:53:53,520] {logging_mixin.py:104} INFO - [2025-02-11 15:53:53,519] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dev/test_1/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dev/test_1/dag.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2025-02-11 15:53:53,521] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:53:53,592] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.085 seconds
[2025-02-11 15:54:23,925] {scheduler_job.py:182} INFO - Started process (PID=132) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:54:23,925] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:54:23,926] {logging_mixin.py:104} INFO - [2025-02-11 15:54:23,926] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:54:23,936] {logging_mixin.py:104} INFO - [2025-02-11 15:54:23,935] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dev/test_1/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dev/test_1/dag.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2025-02-11 15:54:23,936] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:54:24,006] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.082 seconds
[2025-02-11 15:54:55,175] {scheduler_job.py:182} INFO - Started process (PID=164) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:54:55,176] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:54:55,179] {logging_mixin.py:104} INFO - [2025-02-11 15:54:55,179] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:54:55,189] {logging_mixin.py:104} INFO - [2025-02-11 15:54:55,188] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dev/test_1/dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dev/test_1/dag.py", line 8, in <module>
    from airflow.operators.sql import SQLExecuteQueryOperator
ImportError: cannot import name 'SQLExecuteQueryOperator'
[2025-02-11 15:54:55,190] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:54:55,258] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.084 seconds
[2025-02-11 15:55:25,319] {scheduler_job.py:182} INFO - Started process (PID=193) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:55:25,320] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:55:25,320] {logging_mixin.py:104} INFO - [2025-02-11 15:55:25,320] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:55:25,321] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:55:25,321] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.003 seconds
[2025-02-11 15:55:55,449] {scheduler_job.py:182} INFO - Started process (PID=223) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:55:55,450] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:55:55,450] {logging_mixin.py:104} INFO - [2025-02-11 15:55:55,450] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:55:55,452] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:55:55,452] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.005 seconds
[2025-02-11 15:56:25,469] {scheduler_job.py:182} INFO - Started process (PID=251) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:56:25,471] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:56:25,471] {logging_mixin.py:104} INFO - [2025-02-11 15:56:25,471] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:56:25,471] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:56:25,472] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.005 seconds
[2025-02-11 15:56:55,556] {scheduler_job.py:182} INFO - Started process (PID=280) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:56:55,558] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:56:55,559] {logging_mixin.py:104} INFO - [2025-02-11 15:56:55,559] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:56:55,560] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:56:55,560] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.009 seconds
[2025-02-11 15:57:25,628] {scheduler_job.py:182} INFO - Started process (PID=310) to work on /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:57:25,628] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dev/test_1/dag.py for tasks to queue
[2025-02-11 15:57:25,629] {logging_mixin.py:104} INFO - [2025-02-11 15:57:25,629] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:57:25,629] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dev/test_1/dag.py
[2025-02-11 15:57:25,629] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dev/test_1/dag.py took 0.003 seconds
